firebase-appdistribution/firebase-appdistribution.gradle

```diff
@@ -64,6 +67,8 @@ dependencies {
     implementation "androidx.browser:browser:1.3.0"
     implementation "androidx.constraintlayout:constraintlayout:2.1.4"
 
+    implementation "com.google.android.material:material:1.4.0"
```

Comment:
Is this just so we can use `Theme.MaterialComponents.Light.NoActionBar`? If so, could we use `Theme.AppCompat.Light.NoActionBar` instead since we're already depending on appcompat?

Reply:
Done